src/imaCore.mjs

```diff
@@ -1276,23 +1012,26 @@ let gIsOneTransferInProgressInThisThread = false;
 
 export async function doTransfer(
     strDirection, joRuntimeOpts,
-    ethersProviderSrc, joMessageProxySrc, joAccountSrc,
-    ethersProviderDst, joMessageProxyDst, joAccountDst,
+    ethersProviderSrc, joMessageProxySrc, joMessageProxySrcABI, joAccountSrc,
+    ethersProviderDst, joMessageProxyDst, joMessageProxyDstABI, joAccountDst,
     chainNameSrc, chainNameDst, chainIdSrc, chainIdDst,
     joDepositBoxMainNet, // for logs validation on mainnet
     joTokenManagerSChain, // for logs validation on s-chain
     nTransactionsCountInBlock,
     nTransferSteps, nMaxTransactionsCount, nBlockAwaitDepth, nBlockAge,
-    fnSignMessages, joExtraSignOpts, transactionCustomizerDst
+    fnSignMessages, joExtraSignOpts, transactionCustomizerDst, optsChainPair
```

Comment:
Is it still relevant?

Reply:
yes, because `optsChainPair` is passed into event log scanner and printed into log output to let Evg see what he needs

Reply:
Eugene said that urls are enough